src/static/js/header_additional.js

```diff
@@ -599,7 +599,9 @@ additional = {
                 var d = String(t.attr("data-post"));
                 // mandatory additional fields have a post attribute prefixed with a.1
                 if (d.indexOf("a.1") != -1) {
-                    if (common.trim(t.val()) == "") {
+                    if ($("#species").val() && !t.attr("data-speciesids").split(",").includes($("#species").val()) ) {
+                    }
```

Comment:
This doesn't look to be doing anything except I guess making sure that a species specific field can't be blank? I don't like that it's looking for #species anywhere in the DOM too, those kinds of behaviours should be in the caller I think.